.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -0,0 +1,90 @@
+name: Fix #4: Adds Issue Template
+description: Fix #4: Adds Issue Template
```

Comment:
description: Mention details of what this bug is

Reply:
updated this too. thanks @rt4914